apps/aecore/src/aec_chain_state.erl

```diff
@@ -1149,7 +1170,7 @@ maybe_pogf(Node, [Sibling|T]) ->
 % don't award her with mining reward but we lock the excess of coins on the
 % next granting of fees. This way we can compute properly the locked amount.
 calc_rewards(FraudStatus1, FraudStatus2, GenerationFees,
-             K2MineReward, K1FraudReward, IsKey1Genesis) ->
+    K2MineReward, K1FraudReward, IsKey1Genesis) ->
```

Comment:
Wasn't the indentation nicer here before? Overall the indentation is better, but some reduction in indentation may be confusing.